examples/unicorn.yml

```diff
@@ -0,0 +1,18 @@
+# Unicorn process example with RAILS_ENV variable
+
+processes:
+  - name: unicorn
+    config:
+      pid_file: tmp/pids/unicorn.pid
+      start_command: bundle exec unicorn -c config/unicorn/<%= ENV['RAILS_ENV'] %>.rb -D
```

Comment:
We can put the flags together to improve readability a bit:
```yaml
start_command: bundle exec unicorn -Dc config/unicorn/<%= ENV["RAILS_ENV" %>.rb
```

Realistically, this file should be given an `.erb` extension to make it obvious that we're using ERB features here.